uber/automated_emails.py

```diff
@@ -330,7 +332,7 @@ def __init__(self, conf):
 IT WILL CAUSE ANY EMAILS THAT HAVE ALREADY SENT TO RE-SEND.
 ************************************************************************
 
-WORK WE NEED TO DO: Going forward, every single email category below should have an explicit 'ident' set. However,
```

Comment:
Yeah, honestly we cna probably just remove this entire comment block completely since the main point of the whole thing is to warn people about not changing email subjects and to explain the interaction between email subjects and idents, neither of which is any longer a concern.